util.py

```diff
@@ -96,8 +96,14 @@ def sql_alchemy_to_dict(
     sql_alchemy_object: Any,
     for_frontend: bool = False,
     column_whitelist: Optional[Iterable[str]] = None,
+    column_blacklist: Optional[Iterable[str]] = None,
+    column_rename_map: Optional[Dict[str, str]] = None,
 ):
-    result = __sql_alchemy_to_dict(sql_alchemy_object, column_whitelist)
+    result = __sql_alchemy_to_dict(
+        sql_alchemy_object, column_whitelist, column_blacklist
+    )
+    if column_rename_map:
```

Comment:
this would only rename the topmost level. Not sure if intendet but probably not

Reply:
how far should it go, what is the second and third level?